jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -431,19 +431,23 @@ private void writeErrorJson(HttpServletRequest request, PrintWriter writer, int
     {
         writer
             .append("{\n")
-            .append("  url: \"").append(request.getRequestURI()).append("\",\n")
-            .append("  status: \"").append(Integer.toString(code)).append("\",\n")
-            .append("  message: ").append(QuotedStringTokenizer.quote(message)).append(",\n");
+            .append("  \"url\": \"").append(request.getRequestURI()).append("\",\n")
+            .append("  \"status\": \"").append(Integer.toString(code)).append("\",\n")
+            .append("  \"message\": ").append(QuotedStringTokenizer.quote(message)).append(",\n");
         Object servlet = request.getAttribute(Dispatcher.ERROR_SERVLET_NAME);
         if (servlet != null)
             writer.append("servlet: \"").append(servlet.toString()).append("\",\n");
```

Comment:
there is still a ',' at the end if servlet is not null and cause is null, I forgot to mention in the bug report

Reply:
@gbrehmer good catch